pyproject.toml

```diff
@@ -53,7 +53,7 @@ dependencies = [
   "numpy",
   "pandas",
   "matplotlib",
-  "xarray",
+  "xarray[io]",
```

Comment:
Why use `xarray[io]` instead of `h5netcdf` directly? If at some point xarray removes h5netcdf from the io dependencies, zampy would break.

Reply:
good point, it is fixed.